frontend/src/main/scala/bloop/bsp/BloopBspDefinitions.scala

```diff
@@ -1,11 +1,10 @@
 package bloop.bsp
 
```

Comment:
Thanks for removing the wildcard imports, I know it's a pain to get the previous, explicit imports right :+1: